src/backend/core/tests/utils.py

```
@@ -0,0 +1,20 @@
+import json
+
+def assert_equals_partial(actual, expected, debug=False):
+    if debug:
+        print(json.dumps(actual, indent=4), json.dumps(expected, indent=4))
```

Ruff (0.13.2)

`print` found

Remove `print`

(T201)